crates/driver/src/tests/setup/mod.rs

```diff
@@ -839,6 +832,11 @@ impl Setup {
         self
     }
 
+    pub fn auction_id(mut self, auction_id: i64) -> Self {
```

Comment:
Since the testing framework becomes more and more complex a doc comment on when one might use this function would be good.